js/bitmart.js

```diff
@@ -733,7 +733,7 @@ module.exports = class bitmart extends Exchange {
         }
```

Comment:
If the `average` is undefined, we will need to clean up lines 730-733, the calculation of `average` is now done in safeTicker, if `average` is missing.